forge-gui/res/cardsfolder/upcoming/flamewar_brash_veteran_flamewar_streetwise_operative.txt

```diff
@@ -0,0 +1,27 @@
+Name:Flamewar, Brash Veteran
+ManaCost:1 B R
+Types:Legendary Artifact Creature Robot
+PT:3/2
+K:More Than Meets the Eye:B R
+A:AB$ PutCounter | Cost$ Sac<1/Artifact.Other/another artifact> | Defined$ Self | CounterType$ P1P1 | CounterNum$ 1 | SorcerySpeed$ True | SubAbility$ DBConvert | AILogic$ AristocratCounters | SpellDescription$ Put a +1/+1 counter on CARDNAME and convert it.
+SVar:DBConvert:DB$ SetState | Mode$ Convert
+A:AB$ ChangeZone | Cost$ 1 Discard<1/Hand> | ValidCard$ Card.IsRemembered+ExiledWithSource | Origin$ Exile | Destination$ Hand | SpellDescription$ Put all exiled cards you own with intel counters on them into your hand.
```

Comment:
This ValidCard seems wrong – this should be **any** exiled card you own with intel counters on it... so if somehow some other card managed to do that, this should get those, too